packages/salesforcedx-vscode-core/src/commands/util/sfdxCommandlet.ts

```diff
@@ -67,7 +68,7 @@ export abstract class SfdxCommandletExecutor<T>
     const cancellationTokenSource = new vscode.CancellationTokenSource();
     const cancellationToken = cancellationTokenSource.token;
     const execution = new CliCommandExecutor(this.build(response.data), {
-      cwd: getRootWorkspacePath(),
+      cwd: this.executionCwd,
```

Comment:
`getRootWorkspacePath` is not exposed to other packages outside of core, so this was the easiest way to have custom `execute` implementations still have access to this value.